bin/export-record-barcodes

```diff
@@ -55,6 +56,19 @@ FIELDS = [
     *BARCODE_FIELDS,
 ]
 
+OUTPUT_FIELDS = [
+    "project_id",
+    "project_lang",
+    "project_purview",
+    "record_id",
+    "event_name",
+    "repeat_instance",
+    "record_url",
+    "record_link",
```

Comment:
Does the json (that contains commas) in record_link get encoded properly in the CSV so it imports as a single column? If not, you could configure DictWriter to surround entries in quotes. quoting=csv.QUOTE_ALL

Reply:
Yep, it gets quoted properly. The `unix` dialect in use quotes all fields (`QUOTE_ALL`), but the default dialect also quotes as needed (`QUOTE_MINIMAL`).